pom.xml

```diff
@@ -107,6 +107,7 @@
         <version.log4j>2.12.1</version.log4j>
         <version.log4j2-ecs-layout>1.0.1</version.log4j2-ecs-layout>
         <version.spring>5.0.15.RELEASE</version.spring>
+        <version.spring-boot>2.2.2.RELEASE</version.spring-boot>
```

Comment:
not sure if it's worth having such a global version as most plugins use their own. We could align it in another PR if it makes sense.